SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -613,6 +618,26 @@ void USpatialActorChannel::SetChannelActor(AActor* InActor)
 	}
 }
 
+bool USpatialActorChannel::TryAllocateEntityId()
+{
+	EntityId = NetDriver->PackageMap->AllocateEntityIdForActor(Actor);
+
+	if (EntityId != SpatialConstants::INVALID_ENTITY_ID)
```

Comment:
Just to clean up this function, how about we invert this `if` and return, that way the below logic isn't indented as much.